assets/pages/contributions-thankyou/oneoffContributionsThankyou.jsx

```diff
@@ -27,18 +34,23 @@ const content = (
     <section className="thankyou gu-content-filler">
       <div className="thankyou__content gu-content-filler__inner">
         <div className="thankyou__wrapper">
-          <h1 className="thankyou__heading">Thank you!</h1>
+          <h1 className="thankyou__heading">Thank you</h1>
           <h2 className="thankyou__subheading" id="qa-thank-you-message">
             <p>
-              You&#39;ve made a vital contribution that will help us maintain
-              our independent, investigative journalism
+              You&#39;ve made a vital contribution that will help us maintain our independent,
+                investigative journalism.
             </p>
           </h2>
+          <h2 className="thankyou__cta-ask">
+            We need ongoing support from our readers, now more than ever &ndash; show
+              sustained support for the Guardian from as little as {currency.glyph}5 a month
+          </h2>
           <CtaLink
             ctaId="return-to-the-guardian"
-            text="Return to the Guardian"
-            url="https://theguardian.com"
-            accessibilityHint="Go to theguardian dot com front page"
+            text={`Contribute ${currency.glyph}5 a month`}
+            url={`/contribute/recurring?contributionValue=5&contribType=MONTHLY&currency=${currency.iso}&INTCMP=${campaignCode}`}
+            accessibilityHint={`Contribute from ${currency.glyph}5 a month`}
+            onClick={trackComponentEvents('CLICK', campaignCode)}
```

Comment:
For this don't you want a function that, when called, will track the event? E.g.:

```jsx
onClick={() => trackComponentEvents('CLICK', campaignCode)}
```